### PR TITLE
Create_dist should strip on linux 

### PR DESCRIPTION
**Pull request by posix4e**
This will strip and create symbols on linux

### CODE REVIEW

**Comment by posix4e** [comment]
I suppose chromedriver should be stripped as well

File: BUILD.gn [hunk 1]
```diff
@@ -646,6 +646,13 @@ action("create_dist") {
     ]
   }
 
+  if (is_linux) {
+    deps += [ 
+      "//electron/app/linux:strip_and_dump_brave_symbols",
```
Comment (bridiver) [inline BUILD.gn]: this works, but we could keep these inside the linux BUILD.gn if you made them dependencies of `//electron/app/linux:dist_resources`

**Comment by posix4e** [comment]
Thanks for the first review.

**Comment by bridiver** [comment]
++

**Pull request merged**